service/orchestrator/orchestrator.go

```diff
@@ -83,6 +81,7 @@ func init() {
 // ServiceOption is a function-style option to configure the Orchestrator Service
 type ServiceOption func(*Service)
 
+// TODO(all): Function currently not used
 // WithMetricsFile can be used to load a different metrics file
 func WithMetricsFile(file string) ServiceOption {
 	return func(s *Service) {
```

Comment:
```suggestion

```

Reply:
I would keep this function please. I need it in the future :)

Reply:
@oxisto Do we need that?

Reply:
Ok